torch/ao/quantization/quantizer/__init__.py

```diff
@@ -1,35 +1,21 @@
-from .composable_quantizer import ComposableQuantizer
-from .embedding_quantizer import EmbeddingQuantizer
 from .quantizer import (
     DerivedQuantizationSpec,
     EdgeOrNode,
     FixedQParamsQuantizationSpec,
-    OperatorConfig,
-    OperatorPatternType,
     QuantizationAnnotation,
-    QuantizationConfig,
     QuantizationSpec,
     QuantizationSpecBase,
     Quantizer,
     SharedQuantizationSpec,
 )
-from .x86_inductor_quantizer import X86InductorQuantizer
-from .xnnpack_quantizer import XNNPACKQuantizer
 
 __all__ = [
-    "ComposableQuantizer",
```

Comment:
why remove composable quantizer?

Reply:
I think this should not be part of the core quantizer, user can import this through `torch.ao.quantization.composable_quantizer`

Reply:
OK, sounds fine that for us to maintain it. I'm wondering if we want to move xnnpack quantizer, x86 inductor quantizer etc. to a different location since it might be confusing to put them in ao/quantization/quantizer folder because they are not part of the core API